webapp/src/lib/utils.js

```diff
@@ -99,26 +99,25 @@ export function isExpired(expires_at) {
 }
 
 /**
- * Returns if object is Open
+ * Returns if object is Open and not expired
  * @param {object} - obj with status & tx_status fields
  * @param  {string} - status
  * @returns {boolean}
  */
 export function isOpen(obj, status) {
-  return hasStatus(obj, status)
+  return hasStatus(obj, status) && !isExpired(obj.expires_at)
 }
 
 /**
  * Returns if object has status
  * @param {object} - obj with status & tx_status fields
- * @param  {string} - status
+ * @param  {array | string} - status
  * @returns {boolean}
  */
 export function hasStatus(obj, status) {
   return (
     obj &&
-    obj.status === status &&
-    obj.tx_status === txUtils.TRANSACTION_STATUS.confirmed &&
-    !isExpired(obj)
+    status.includes(obj.status) &&
```

Comment:
This shouldn't matter in the long run, buuut, if you have an status named `open` and another named `opened` then:

```javascript
  hasStatus({ status: 'open' }, 'opened') // true
```

I don't think it's a really a big deal, but wanted to point it out just in case

Reply:
Cool, what do you think of refactoring it once we need it?